src/Sulu/Bundle/AdminBundle/Resources/js/services/DOM/afterExecutionQueue.js

```diff
@@ -0,0 +1,4 @@
+// @flow
+export default function afterExecutionQueue(callback: () => *) {
```

Comment:
Shouldn't we call it `waitForElementRender` instead? The name already tells pretty much how this is solved. If we find a better solution one day we would have to rename this method and introduce a BC break...